init.py

```diff
@@ -1,339 +1,22 @@
-    target = os.path.join(target_dir, 'scale_free_graph_%s_%s') % (n, k)
-    for v in g:
-        for to in g[v]:
-            add_rdf_edge(v, to[1], to[0], output_graph)
-
-    write_to_rdf(target, output_graph)
-
-
-def clean_dir(path):
-    if os.path.isdir(path):
-        shutil.rmtree(path)
-    os.mkdir(path)
-
-
-def add_graph_dir(name):
-    matrices_dir = os.path.join(DATA_ROOT_DIR, name, MATRICES_DIR)
-    clean_dir(matrices_dir)
-    return matrices_dir
-
-
-def generate_all_sparse_graphs():
-    print('Sparse graphs generation is started.')
-    add_graph_dir('SparseGraph')
-    matrices_dir = add_graph_dir('SparseGraph')
-    for g in SPARSE_GRAPH_TO_GEN:
-        gen_sparse_graph(matrices_dir, g[0], g[1])
-    print('Sparse graphs generation is finished.')
-
-
-def generate_full_graphs():
-    print('Full graphs generation is started.')
-    matrices_dir = add_graph_dir('FullGraph')
-    for g in FULL_GRAPH_TO_GEN:
-        gen_cycle_graph(matrices_dir, g)
-    print('Full graphs generation is finished.')
-
-
-def generate_worst_case_graphs():
-    print('Worst case graphs generation is started.')
-    matrices_dir = add_graph_dir('WorstCase')
-    for n in range(2, NUMBER_OF_WORST_CASES):
-        gen_worst_case_graph(matrices_dir, 2 ** n)
-    print('Worst case graphs generation is finished.')
-
-
-def generate_scale_free_graphs():
-    print('Scale free graphs generation is started.')
-    matrices_dir = add_graph_dir('ScaleFree')
-    for k in 1, 3, 5, 10:
-        for n in 100, 500, 2500, 10000:
-            gen_scale_free_graph(matrices_dir, n, k, ['a', 'b', 'c', 'd'])
-    print('Scale free graphs generation is finished.')
-
-
-def gen_sierpinski_graph(target_dir, degree, predicates=['A']):
-    ''' Generates a Sierpinski Triangle graph. '''
-
-    def sierpinski(t, l, r, deg, preds, g):
-        ''' Core function for generating the Sierpinski Triangle. '''
-        if deg > 0:
-            lt = next(ids)
-            tr = next(ids)
-            rl = next(ids)
-            sierpinski(l, lt, rl, deg - 1, preds, g)
-            sierpinski(lt, t, tr, deg - 1, preds, g)
-            sierpinski(rl, tr, r, deg - 1, preds, g)
-        else:
-            add_edges(l, t, preds, g)
-            add_edges(t, r, preds, g)
-            add_edges(r, l, preds, g)
-
-    def add_edges(u, v, preds, g):
-        ''' Adds edges between vertices u and v for all predicates. '''
-        for p in preds:
-            g += [[u, p, v]]
-            g += [[v, p, u]]
-
-    def _idgen():
-        ''' Generates integer identifiers for vertices. '''
-        c = 4
-        while True:
-            yield c
-            c += 1
-
-    ids = _idgen()
-    graph = []
-    sierpinski(1, 2, 3, degree, predicates, graph)
-    with open(
-        os.path.join(target_dir, 'sierpinskigraph_%s.txt' % (degree)), 'w'
-    ) as out_file:
-        for triple in graph:
-            out_file.write('%s %s %s \n' % (triple[0], triple[1], triple[2]))
+#!/usr/bin/python3
```

Comment:
Seems, that script is not only for initialization from now. Should it be renamed?

Reply:
Yes, I wanted to rename it, but I didn't come up with anything. Can you suggest a better name?

Reply:
No, I cannot. Maybe just ```main```.